config/tests/accessor_test.go

```diff
@@ -458,38 +462,6 @@ func TestAccessor_UpdateConfig(t *testing.T) {
 			assert.True(t, called)
 		})
 
-		t.Run(fmt.Sprintf("[%v] Change handler on change", provider(config.Options{}).ID()), func(t *testing.T) {
```

Comment:
why delete this test?

Reply:
Because a change went into go 1.8 that affected how fsnotify can monitor file system changes that this no longer works... the way k8s mounts and replaces contents of config maps though is tested in the subsequent test and that continues to pass.